common/decklist.cpp

```diff
@@ -336,7 +336,7 @@ DeckList::DeckList()
     root = new InnerDecklistNode;
 }
 
-// TODO: http://qt-project.org/doc/qt-4.8/qobject.html#no-copy-constructor-or-assignment-operator
+// TODO: https://qt-project.org/doc/qt-4.8/qobject.html#no-copy-constructor-or-assignment-operator
```

Comment:
this is a dead link

Reply:
https://doc.qt.io/qt-5/qobject.html#no-copy-constructor-or-assignment-operator if you wanted to update this comment, I'm not sure what the problem is though.

Reply:
It'll auto redirect

Reply:
no, because the qt-project.org domain uses an old cert it seems